### DISCUSSION TITLE
fcitx5 on i3

### DISCUSSION THREAD

**Discussion by baoke344** [fcitx/fcitx5#783] [Q&A]
Hi! I am using i3 and installed fcitx5
I have fcitx5 auto started by putting this in my i3 config 
`exec --no-startup-id fcitx5 -d`
fcitx5 running fine
added some config using fcitx5-configtool but when press `ctrl+space` not thing happened 
Right click on the system tray also nothing

How can I fix this?

**Comment by wengxt** [comment]
Please post the output of fcitx5-diagnose
